levelkdev/swapr-dapp
# Add routing logos for DFYN and QuickSwap to the Routing Through section.

# Summary

Fixes related to #906 
Adds DFNY and QuickSwap logos to Routing Through Section.

![image](https://user-images.githubusercontent.com/36545016/165324312-cbfaba83-cd8b-4d3b-9c49-1f998ae5aed3.png)
|:--:|
| <b>Image - On localhost with SDK in branch</b>|

  # To Test/Replicate

1. Follow the procedure to link the SDK locally (see below)
2. `yarn start`
3. Open on your browser and scroll down

### IMPORTANT
This PR uses unreleased SDK.

TL;DR
To make it work you need to build & link [this SDK branch](https://github.com/levelkdev/swapr-sdk/tree/feature/add-polygon-config) locally.

How to do that: (from [relevant PR](https://github.com/levelkdev/swapr-dapp/pull/910))

Clone SDK repo locally
Checkout to `feature/add-polygon-config` on the SDK repo
run `yarn prepublishOnly`
run `yarn link`
cd to swapr-dApp and run `yarn link @swapr/sdk`
All types/missing imports should work fine now.

After you're done playing with linked repo you need to unlink local build. To do this, in your swapr-dApp `run yarn unlink @swapr/sdk && yarn install --force`
